### PR TITLE
Reduce action space, add 8x8 stripes, option to scale tinyworld obs

### PR DESCRIPTION
**Pull request by taufeeque9**
This PR adds the following features:
- `tinyworld_scale` argument that scales tiny world obs from 1x1 to NxN (used only when `tinyworld_obs=True`)
- `render_mode="rgb_8x8"` that uses the 8x8 stripes from the Guez et al. paper
- `create_stripes.ipynb` to create the stripes from the Guez et al. paper
- Reduces action space to only contain 4 actions

### CODE REVIEW

File: gym_sokoban/envs/surface/8x8/create-stripes.ipynb
Comment (rhaps0dy) [inline gym_sokoban/envs/surface/8x8/create-stripes.ipynb:1]: Omg these are beautiful 😍

File: gym_sokoban/envs/sokoban_env.py [hunk 1]
```diff
@@ -83,11 +90,8 @@ def step(self, action):
...
 
-        if action == 0:
-            moved_player = False
-
         # All push actions are in the range of [0, 3]
-        elif action < 5:
+        if action < 5:
             moved_player, moved_box = self._push(action)
 
         else:
```
Comment (rhaps0dy) [inline gym_sokoban/envs/sokoban_env.py]: This `else` branch should not exist anymore

Comment (rhaps0dy) [reply gym_sokoban/envs/sokoban_env.py]: And neither should the `_move` method I think

File: gym_sokoban/envs/sokoban_env.py [hunk 2]
```diff
@@ -83,11 +90,8 @@ def step(self, action):
 
         moved_box = False
 
-        if action == 0:
-            moved_player = False
-
         # All push actions are in the range of [0, 3]
-        elif action < 5:
+        if action < 5:
```
Comment (rhaps0dy) [inline gym_sokoban/envs/sokoban_env.py]: This will always be true, skip the check

File: gym_sokoban/envs/sokoban_env.py [hunk 3]
```diff
@@ -119,7 +123,7 @@ def _push(self, action):
         :param action:
         :return: Boolean, indicating a change of the room's state
         """
-        change = CHANGE_COORDINATES[(action - 1) % 4]
+        change = CHANGE_COORDINATES[(action) % 4]
```
Comment (rhaps0dy) [inline gym_sokoban/envs/sokoban_env.py]: Actions are all smaller than 4
```suggestion
        change = CHANGE_COORDINATES[action]
```

and let it fail with IndexError if it's the wrong number

File: gym_sokoban/envs/sokoban_env.py [hunk 4]
```diff
@@ -160,7 +164,7 @@ def _move(self, action):
...
```
Comment (rhaps0dy) [inline gym_sokoban/envs/sokoban_env.py]:
```suggestion
        change = CHANGE_COORDINATES[action]
```

like above

**Review by rhaps0dy — APPROVED** [review]
LGTM but please address the feedback, esp the `% 4` stuff

File: gym_sokoban/envs/sokoban_env.py (hunk 4 above)
Comment (taufeeque9) [reply gym_sokoban/envs/sokoban_env.py]: removed _move function

**Pull request merged**